crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -340,14 +340,11 @@ def prepare_nodes_for_os_upgrade
       prop.save
     end
 
-    # Change pacemaker setting to not use pacemaker for stateless services.
-    # Also, turn off possible DRBD enablement.
+    # Turn off possible DRBD enablement.
```

Comment:
I wonder if we should remove all this DRBD related code soon.

Reply:
The attributes are still there so I left this part intact.